GetJobber/atlantis
# feat(components): add 'pinned' and 'unPinned' icons — (Minor) Feature Release

## Motivations

We need these icons for our pinned Note work over on Work Setup

## Changes



### Added

- Added `pinned` and `unPinned` svgs to icons

## Testing

Following the testing steps from [this guide](https://jobber.atlassian.net/wiki/spaces/DES/pages/1658585445/Technical+Guide+Adding+an+icon+to+Atlantis):
- Pull the branch
- `npm run bootstrap`
- `npm start`
- Check to ensure the icons are included in our storybook for `Icon`

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

![Random photo of Atlantis](https://loremflickr.com/672/400/atlantis)
